docs/CreateKoblenzHash.md

```diff
@@ -0,0 +1,62 @@
+# Creating Hashes for Koblenz Pass Data
+
+## Steps
+
+The example data is
+
+| Parameter    | Value         |
+|--------------|---------------|
+| Name         | Karla Koblenz |
+| Geburtstag   | 10.06.2003    |
+| Aktenzeichen | 123K          |
+
+
+### 1. Collect all data and merge it into an object
+```agsl
+//Example:
+{
+    full_name: "Karla Koblenz"
+    birthday: 12213
+    referenceNumber: "123K"
+}
+```
+
+- The full_name must be `FirstnameSpaceLastname`. Every char must exaclty match the user input, as otherwise there is not possiblity to match the data Koblenz transfers with the input the user makes.
+e.g. `Karla Koblenz` will match neither with `Karla Lisa Koblenz` nor with `Karlá Koblenz`.
+- The birthday is defined in our protobuf [card.proto](../frontend/card.proto) file: It counts the days since the birthday (calculated from 1970-01-01).
+  All values of this field are valid, including the 0, which indicates that the birthday is on 1970-01-01. Birthdays before 1970-01-01 have negative values.
```

Comment:
I see your point, would keep it like this, so at least our code is consistent?

Reply:
we use it this standard (dd.MM.yyyy) for query params (used by 3rd party applications from nuernberg) and csv import. I'm with steffi in this case

Reply:
ISO 8601 might be problematic because there are maybe multiple encodings of the same date. Not sure though. I'm just thinking about the "+Z" and "+00:00"

Reply:
i'm gonna keep the calculated birthday here if thats okay for everyone